specs/tilebag_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/tilebag'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'TileBag' do
+  describe 'initialize' do
+    it 'creates and instance'do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.must_be_instance_of Scrabble::TileBag
+    end
+
+    it 'contains a collection of tiles' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.default_tiles.must_be_instance_of Hash
+    end
+
+    it 'returns 98 tiles ' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.default_tiles.values.sum.must_equal 98
+    end
+
+  describe 'draw_tiles(num)' do
+    it 'returns a collection of random tiles' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.draw_tiles(4).must_be_instance_of Array
+    end
```

Comment:
You should check more than that it gives back an array here. For example:

- Is the array the correct length?
- Are all the members letters?
- Are the letters that were returned the same as the ones that were removed from the bag (how would you test this?)

Expanding your tests in this way would have caught that bug with draw_tiles.